install/on_rpi/on_pi_setup.sh

```diff
@@ -40,6 +40,8 @@ sudo cp $HOME/bthidhub/install/on_rpi/sdp_record.xml /etc/bluetooth/sdp_record.x
 sudo cp $HOME/bthidhub/install/on_rpi/input.conf /etc/bluetooth/input.conf
 sudo cp $HOME/bthidhub/install/on_rpi/main.conf /etc/bluetooth/main.conf
 
+sudo sed -i "s/\/home\/pi/\/home\/$(basename $HOME)/g" $HOME/bthidhub/install/on_rpi/remapper.service
```

Comment:
Perhaps it'd be a little safer to just put `{pi_user}` in the service file and replace that?
Also, I'd rather edit the copied file than end up with uncommitted changes in the repo (i.e. regular UI improvements can upgraded with `git pull` if the repo contents are not modified).